docs/src/modules/java/pages/kickstart.adoc

```diff
@@ -165,6 +165,55 @@ The Maven failsafe plugin runs the integration tests when the `it` profile is en
 mvn verify -Pit
 ----
```

Comment:
Above is a FIXME to include the CounterTest.java, can you include that again. It should only show the generated stub.

Reply:
I rollback to previous version